Dockerfile

```diff
@@ -0,0 +1,66 @@
+# Description: Base image for ColumnFlow
+# Version: 1.0
+# Created by: P. Keicher
+
+# Arguments
+# base: Base image. Can either be
+#       - a Python image for the most basic setup or
+#       - a custom image with the basic software stack for cf
+ARG pyversion=""
+ARG base="python:$pyversion"
+FROM $base
+
+USER root
+
+ARG pyversion=""
```

Comment:
This is actually need afaik because all arguments defined before the `FROM` statement are lost after it. Since `pyversion` is needed both before and after the statement, it needs to be loaded twice (which is somewhat stupid)